src/main/sensors/battery.c

```diff
@@ -67,6 +67,13 @@ uint16_t batteryWarningVoltage;
 uint16_t batteryCriticalVoltage;
 static lowVoltageCutoff_t lowVoltageCutoff;
 //
+
+// Battery delay state stuff
+uint16_t start_t = 0;
+uint16_t end_t = 0;
+uint16_t total_t;
```

Comment:
`millis()` returns `timeMs_t`, which is 32 bit. Using `uint16` here will cause an overflow, and undefined behaviour, 65 seconds after power up.
Also, please make these static if they aren't exported.
And please use proper camelCase.

Reply:
It would be probably cleaner to to sample `millis()` every time voltage is above threshold (`mstime_t lastVoltageOkMillis` ?). Then use `cmp32` to get delta for comparison.
Only one static variable is will be necessary.

Also multiplication is faster, so multiply user supplied value instead of dividing delta time.